src/components/Proposal/ProposalSummary/ProposalSummaryStandardBounties.tsx

```diff
@@ -0,0 +1,276 @@
+import { IProposalState } from "@daostack/client";
+import * as classNames from "classnames";
+import { GenericSchemeInfo } from "genericSchemeRegistry";
+import { linkToEtherScan } from "lib/util";
+import * as React from "react";
+import * as css from "./ProposalSummary.scss";
+
+const web3 = require("web3");
+
+interface IProps {
+  genericSchemeInfo: GenericSchemeInfo;
+  detailView?: boolean;
+  proposal: IProposalState;
+  transactionModal?: boolean;
+}
+
+export default class ProposalSummaryStandardBounties extends React.Component<IProps, null> {
+
+  public render(): RenderOutput {
+    const { proposal, detailView, genericSchemeInfo, transactionModal } = this.props;
+    let decodedCallData: any;
+    try {
+      decodedCallData = genericSchemeInfo.decodeCallData(proposal.genericScheme.callData);
+    } catch(err) {
+      if (err.message.match(/no action matching/gi)) {
+        return <div>Error: {err.message} </div>;
+      } else {
+        throw err;
+      }
+    }
+    const action = decodedCallData.action;
+
+    const proposalSummaryClass = classNames({
+      [css.detailView]: detailView,
+      [css.transactionModal]: transactionModal,
+      [css.proposalSummary]: true,
+      [css.withDetails]: true,
+    });
+
+    switch (action.id) {
+      case "issueAndContribute":
+        return (
+          <div className={proposalSummaryClass}>
+            <span className={css.summaryTitle}>
+              <img src="/assets/images/Icon/edit-sm.svg" />&nbsp; {action.label}
+            </span>
+            { detailView ?
+              <div className={css.summaryDetails}>
+                Bounty Details: <a href={`https://ipfs.io/ipfs/${decodedCallData.values[3]}`} target="_blank" rel="noopener noreferrer">{decodedCallData.values[3]}</a>. <br/>
```

Comment:
Our best practice here is to use divs, not `<br/>`

Reply:
8fa6cde